src/vector/index.html

```diff
@@ -60,7 +60,7 @@
   </head>
   <body style="height: 100%; margin: 0;">
     <noscript>Sorry, Element requires JavaScript to be enabled.</noscript> <!-- TODO: Translate this? -->
-    <section id="matrixchat" style="height: 100%;" class="notranslate"></section>
+    <section id="matrixchat" style="height: 100%; overflow-y: auto" class="notranslate"></section>
```

Comment:
We shouldn't be applying any rules here beyond what is necessary, especially as whole-page scrolling for what is nominally a full screen app is a bad time. The rule should be applied to the smallest effect needed, in the very components it affects. This will likely also break overscroll bounce on OSX and iOS as per the comment in the style file. Such a change could also really benefit from Percy testing.

![image](https://user-images.githubusercontent.com/2403652/201909466-f4328899-07bd-4229-bba5-e9f21e5999eb.png)